src/test/java/com/jcabi/github/mock/MkReleaseAssetsTest.java

```diff
@@ -111,6 +114,24 @@ public void fetchesRelease() throws Exception {
         );
     }
 
+    /**
+     * Must encode the input bytes into Base64.
+     *
```

Comment:
@llorllale remove this newline and same as above, the "Unexpected" sounds strange

Reply:
@amihaiemil ok